.github/pull_request_template.md

```diff
@@ -10,6 +10,29 @@ _Related PRs:_
 - [ ] New feature
 - [ ] Minor changes
 
+## Running the demo
+
+In order to run the included demo app, run `yarn run bootstrap` and then `cd demo` and run both `npx react-native run-android` and `npx react-native start` in parallel.
+
+## Review checklist for new or updated modules
+
+[ ] I have made no modules changes - ignore this section questions in that case
+[ ] I have included a `meta.json` file in the module's root directory
+[ ] I have included a `preview.png` image in the module's root directory
+[ ] `yarn run parse` checks pass
+[ ] I updated the modules build (`/dist` directory) with `yarn run parse --write`
+[ ] `yarn run demo` and `yarn run add app-menu <your-module>` runs correctly
+[ ] **"Running the demo"** app shows the App Menu screen with an entry for your module
+[ ] Clicking your module entry in the menu loads your module's initial screen correctly
```

Comment:
This would be for RN modules, for Django a draft could be:
- [ ] I have checked that the modules' urls are resolved correctly
- [ ] I have checked that the migrations are detected

Reply:
Thanks for the suggestion! Could you please check if 5570227 looks good?

Reply:
Yes, I forgot to add the admin, otherwise looks good!